### DISCUSSION TITLE
Seeking how to obtain MMF groundwater parameters

### DISCUSSION THREAD

**Discussion by zhengzhenk** [NASA-LIS/LISF#1441] [Q&A] [answered]
I am currently looking to utilize the MMF groundwater scheme in the noahmp.4.0.1. In the LDT process, I require the usage of five MMF parameters: FDEPTH, RECHLIM, RIVERBED, EQWTD, and HGT_M. I've noticed that  in issue #1038, it was mentioned these parameter files are available on Discover at the following location: 

`/discover/nobackup/projects/lis/LS_PARAMETERS/noahmp401_parms/groundwater/`

Unfortunately, I do not possess an NCCS Discover high-performance computing account, which is preventing me from accessing these parameters. I inquire whether it is possible for you to provide these parameters on GitHub, or if you could offer guidance on how to generate them. Any suggestions will be greatly appreciated.
@smahanam  and @dmocko ,
Thank you!

**Comment by dmocko** [comment, marked as answer]
Hi @zhengzhenk 

I have copied the required parameters to our LIS data portal.  They are located here:
https://portal.nccs.nasa.gov/lisdata_pub/data/PARAMETERS/noahmp401_parms/groundwater/

**Reply by zhengzhenk** [reply]
Hi @dmocko 
Thank you very much, I will go to the LIS data portal to download the relevant parameters!

**Reply by dmocko** [reply]
Note that the data was originally obtained from this location:
https://www2.mmm.ucar.edu/wrf/users/download/get_sources_wps_geog.html

**Reply by zhengzhenk** [reply]
Thank you very much for your reminder.
